packages/plugin-core/src/_extension.ts

```diff
@@ -1005,7 +1005,7 @@ async function _setupCommands(
       vscode.commands.registerCommand(
         DENDRON_COMMANDS.RELOAD_INDEX.key,
         sentryReportingCallback(async (silent?: boolean) => {
-          const out = await new ReloadIndexCommand().run({ silent });
+          const out = await new ReloadIndexCommand(ws).run({ silent });
```

Comment:
nit: we are passing the extension instance explicitly to ReloadIndexCommand while `ReloadIndexCommand` command does not have a constructor that accepts extension. Even though this will compile in TS a bit weird to pass something explicitly to this instance if it doesn't accept it yet.